app.py

```diff
@@ -1,33 +1,33 @@
-import json
-from functools import wraps
-from flask import Flask, render_template, redirect, session
-
-import auth
 from auth import auth_bp
+from user import user_bp
+from decorators import *
+from db import db, User
+from flask import Flask, render_template, redirect, session, request, url_for, make_response, flash
+from flask_sqlalchemy import SQLAlchemy
+import json
 
 app = Flask(__name__)
-
 app.config.from_file("config.json", load=json.load)
 
-# Core blueprints
-app.register_blueprint(auth_bp, url_prefix="/auth")
+with open('config.json') as config_file:
```

Comment:
The line above already parses the configuration file and save it into the `app.config` dict. You only need to put `SQLALCHEMY_DATABASE_URI` into that file.

Reading your diff before submitting it could help you prevent such code smell.

Reply:
Yes, it's my fault, I wanted to go too fast.